framework/math/sparse_matrix/sparse_matrix.cc

```diff
@@ -218,10 +215,7 @@ void
 SparseMatrix::CheckInitialized() const
 {
   if (rowI_values.empty())
-  {
-    log.LogAllError() << "Illegal call to unitialized SparseMatrix matrix.";
-    Exit(EXIT_FAILURE);
-  }
+    throw std::runtime_error("SparseMatrix: Illegal call to unitialized matrix");
```

Comment:
Can you also fix the typo here?
"unitialized" -> "uninitialized"

Reply:
Fixed.